srcpkgs/fcft/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'fcft'
 pkgname=fcft
-version=3.0.1
+version=3.1.1
```

Comment:
Would you mind updating to .2?

Reply:
Just pushed the new version. I'm testing it right now and it's working well.